src/components/searchableTokenList/PreSuggestedToken.tsx

```diff
@@ -0,0 +1,49 @@
+import { useAppSelector } from 'redux/index';
+import { useMemo } from 'react';
+import { Token } from 'services/observables/tokens';
+
+type PreSuggestedTokenProsp = { onSelect: (token: Token) => void };
+
+const preSuggestedTokenList = ['BNT', 'ETH', 'WBTC', 'USDC', 'USDT'];
+
+export const PreSuggestedToken = ({ onSelect }: PreSuggestedTokenProsp) => {
+  const tokens = useAppSelector<Token[]>((state) => state.bancor.tokens);
```

Comment:
As this list of pre-suggested token is derived from redux state (tokens) and a hard coded list - I don't see why it shouldn't be a redux selector. This is a derived state. We can keep this component logic-free.